<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>416</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>LED 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>160</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>LED 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>160</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>LED 3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>160</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>LED 4</string>
    </property>
   </widget>
   <widget class="LED" name="led">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="LED" name="led_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="LED" name="led_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>50</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="LED" name="led_4">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>50</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>416</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LED</class>
   <extends>QWidget</extends>
   <header>LED.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
